### PR TITLE
Create example-checkout-curl.php

### PR DESCRIPTION
**Pull request by goblinhorde**
Example checkout page for Square Checkout w/o SDK

### CODE REVIEW

File: connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?php
+// Include the Square Connect API resources, store config, and helper functions
+require_once '/local/path/to/sdk/SquareConnect/autoload.php';
```
Comment (bunnyc1986) [inline connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php]: is this still needed?

Comment (goblinhorde) [inline connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php]: Yes. This (and the helper file) are linkd in the setup guide so we're not totally ignoring ppl not using the SDK

Comment (bunnyc1986) [inline connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php]: I'm afraid this could be misleading since the file name says "nosdk".

Comment (goblinhorde) [inline connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php]: What do you suggest as an alternate name?

Comment (goblinhorde) [inline connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php]: Changed the file name. Let me know what you think.

Comment (bunnyc1986) [inline connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php]: looks good! the actual file name needs to be updated too.

Comment (goblinhorde) [inline connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php]: I thought that's what I did =\

**Comment by bunnyc1986** [comment]
lgtm!

**Comment by goblinhorde** [comment]
Changed for reals this time (I think!?)

**Pull request merged**